### PR TITLE
Task/689  add edit option inside the application view page has been …

### PR DESCRIPTION
**Pull request by jananikuruntham**
…added

### CODE REVIEW

File: api/src/applications/applications.service.ts [hunk 1]
```diff
@@ -111,7 +111,15 @@ export class ApplicationsService {
         where: { application: application },
       });
       if (existingApplication) {
-        throw new UnprocessableEntityException('application already exists.');
+        // throw new UnprocessableEntityException('application already exists.');
```
Comment (vsaravanakumar) [inline api/src/applications/applications.service.ts:114]: Remove the commented line

File: front/src/app/applications/[id]/page.tsx [hunk 2]
```diff
@@ -14,27 +14,84 @@ import {
...
+      const response = await ApplicationApi.updateApplication(
+        applicationId,
+        updatedData
+      );
+      setUniqueAlert("");
+      if (response) {
+        if (response.statusCode === 409) {
+          setUniqueAlert(response.setAlertShowmessage);
+        } else if (response.statusCode === 200) {
+          console.log('response.data', response.data);
```
Comment (vsaravanakumar) [inline front/src/app/applications/[id]/page.tsx:77]: Remove console.log

File: front/src/app/groups/[id]/page.tsx [hunk 3]
```diff
@@ -322,6 +340,65 @@ const GroupView = ({ params }: { params: IGroupView }) => {
...
+      } catch (error) {
+        console.error("Error updating group:", error);
+      }
+    }
+  };
+
+  const editGroup = async (id: any, updatedData: any) => {
+    try {
+      const response = await GroupsApi.updateGroupApi(id, updatedData);
+      console.log("res", response);
```
Comment (vsaravanakumar) [inline front/src/app/groups/[id]/page.tsx:364]: Remove console.log

File: front/src/app/groups/[id]/page.tsx [hunk 4]
```diff
@@ -322,6 +340,65 @@ const GroupView = ({ params }: { params: IGroupView }) => {
...
+          });
+        }
+      }
+    } catch (error: any) {
+      var response = error.response.data;
+      if (response.statusCode == 422 && response.message.name) {
+        setUniqueAlert(response.message.name);
+      }
+      if (response.statusCode === 409) {
+        console.log("error");
```
Comment (vsaravanakumar) [inline front/src/app/groups/[id]/page.tsx:381]: Remove console.log

File: api/src/applications/applications.service.ts (hunk 1 above)
Comment (jananikuruntham) [reply api/src/applications/applications.service.ts:114]: commented line was removed

File: front/src/app/applications/[id]/page.tsx (hunk 2 above)
Comment (jananikuruntham) [reply front/src/app/applications/[id]/page.tsx:77]: removed console log

File: front/src/app/groups/[id]/page.tsx (hunk 3 above)
Comment (jananikuruntham) [reply front/src/app/groups/[id]/page.tsx:364]: removed console log

File: front/src/app/groups/[id]/page.tsx (hunk 4 above)
Comment (jananikuruntham) [reply front/src/app/groups/[id]/page.tsx:381]: removed console log

**Pull request merged**